modules/update-preparing-conditional.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
```

Comment:
I am curious as to how this is considered a procedure module? It almost seems like a concept module to me. I was looking at the mod doc guidelines https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

Reply:
This is in fact a conceptual module and not a procedural one, thanks for catching this! Now to fix this error in main as well 🤦‍♂️